dist/tools/uncrustify/uncrustify.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+RIOTBASE=$(git rev-parse --show-toplevel)
+CURDIR=$(cd "$(dirname "$0")" && pwd)
+BLACKLIST=$CURDIR/blacklist.txt
+
+. "$RIOTBASE"/dist/tools/ci/changed_files.sh
+FILES=$(changed_files | grep -vf "$BLACKLIST")
```

Comment:
changed files allows it's own blacklist. Sth like this should be equivalent:

```FILES=$(EXCLUDE="(.*/vendor/.*|.*/core/.*)" changed_files)```

Reply:
I proposed to provide a separate blacklist.txt file because I think it's easier to add/remove entries from there. Since there might be several blacklisted entries, I thought it was a good idea. What do you think?

Reply:
> What do you think?

Ok! Then I'd suggest removing "vendor", as that should be filtered by changed_files' defaults.
Anyone reading this, next one who needs a blacklist file needs to add generic support for that to changed_files.sh! ;)

Reply:
Ok, perfect! ;)